### PR TITLE
Improve the lanes docs

### PR DESCRIPTION
**Pull request by janpio**
depends on #14746 being merged

### CODE REVIEW

File: docs/advanced/lanes.md [hunk 1]
```diff
@@ -92,7 +93,7 @@ lane :calculate do |options|
 end
 ```
 
-## Stop executing a lane early
+# Stop executing a lane early
```
Comment (janpio) [inline docs/advanced/lanes.md]: This is not a child of `Switching lanes` any more.

File: docs/advanced/lanes.md [hunk 2]
```diff
@@ -130,7 +131,7 @@ end
 
 When you stop executing a lane early with `next`, any `after_each` and `after_all` blocks you have will still trigger as usual :+1:
 
-## `before_each` and `after_each` blocks
+# `before_each` and `after_each` blocks
```
Comment (janpio) [inline docs/advanced/lanes.md]: This is not a child of `Switching lanes`.

File: docs/advanced/lanes.md [hunk 3]
```diff
@@ -212,7 +196,24 @@ lane_context[SharedValues::GRADLE_FLAVOR]               # Generated by `gradle`
 lane_context[SharedValues::GRADLE_BUILD_TYPE]           # Generated by `gradle`
 ```
 
-To get information about available lane variables, run `fastlane action [action_name]`.
+To get information about available lane variables, run `fastlane action [action_name]` or look at the generated table in the action documentation.
+
+# Lane Properties
```
Comment (janpio) [inline docs/advanced/lanes.md]: Have to know about `lane_context` first before being able to introduce the properties.

File: docs/advanced/lanes.md [hunk 4]
```diff
@@ -212,7 +196,24 @@ lane_context[SharedValues::GRADLE_FLAVOR]               # Generated by `gradle`
 lane_context[SharedValues::GRADLE_BUILD_TYPE]           # Generated by `gradle`
 ```
 
-To get information about available lane variables, run `fastlane action [action_name]`.
+To get information about available lane variables, run `fastlane action [action_name]` or look at the generated table in the action documentation.
```
Comment (janpio) [inline docs/advanced/lanes.md]: https://github.com/fastlane/fastlane/pull/14746 introduces this table

File: docs/advanced/lanes.md [hunk 5]
```diff
@@ -169,26 +170,9 @@ lane :upload do
...
-
-```ruby
-ENV["FASTLANE_PLATFORM_NAME"]
-ENV["FASTLANE_LANE_NAME"]
-```
-
 # Lane Context
 
-The different actions can *communicate* with each other using a shared hash. You can access this in your lanes with the following code:
+The different [actions](/actions) can *communicate* with each other using a shared hash. You can access this in your code (lanes, actions, plugins etc.):
```
Comment (janpio) [inline docs/advanced/lanes.md]: Details on where you can use this

**Review by rogerluan — APPROVED** [review]
https://github.com/fastlane/fastlane/pull/14746 was merged and this PR LGTM! 🎉

**Pull request merged**